pkgs/development/tools/misc/terraform-plugin-docs/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "terraform-plugin-docs";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = "terraform-plugin-docs";
+    rev = "v${version}";
+    sha256 = "1nwvi3nb4hxpa8pz0zm41yp7skvfafrl313n6zplr0lcs7my5xp6";
+  };
+
+  subPackages = [ "cmd/tfplugindocs" ];
+  vendorSha256 = "AjW6BokLVDkIWXToJ7wNq/g19xKTAfpQ/gVlKCV5qw0=";
```

```suggestion
    hash = "sha256-5vbi69GMgkzvN3aEQbNTbk99rg+kfvAvUrdDsuyIm9s=";
  };

  subPackages = [ "cmd/tfplugindocs" ];
  vendorHash = "sha256-AjW6BokLVDkIWXToJ7wNq/g19xKTAfpQ/gVlKCV5qw0=";
```

I'm not sure, but I think `sha256` / `vendorSha256` are deprecated in favor of `hash` / `vendorHash`.